iocore/net/SSLCertLookup.cc

```diff
@@ -404,6 +389,7 @@ SSLContextStorage::lookup(const char *name) const
 {
   InkHashTableValue value;
 
+  // First look for an exact name match
```

Comment:
The case insensitive lookups came in a separate PR.  Need to rebase this branch to include that.